core/indigo-core/reaction/src/base_reaction_substructure_matcher.cpp

```diff
@@ -120,7 +120,7 @@ bool BaseReactionSubstructureMatcher::find()
             int mode = _matchers.top()->getMode();
 
             //_matchers.reserve(_matchers.size() + 1);
-            AutoPtr<_Matcher> top_matcher(new _Matcher(*_matchers.top()));
+            std::unique_ptr<_Matcher> top_matcher(new _Matcher(*_matchers.top()));
```

Comment:
I think it's better to use `make_unique` in all such cases, avoiding `new` where it's possible.